lib/Controller/PageController.php

```diff
@@ -149,20 +149,6 @@ public function index(): TemplateResponse {
 
 		$this->initialState->provideInitialState('nomedia-paths', $paths);
 
-
-		$key = $user->getUID();
-		$tagCounts = $this->tagCountsCache->get($key);
```

Comment:
AFAIK, this is set up in the constructor. Could you remove tagCountsCache and tagManager there as well?